qc2md.py

```diff
@@ -100,6 +100,18 @@ def parse_args() -> argparse.Namespace:
         default=True,
         help="Display a picker interface if there are multiple matching reference lines (default: %(default)s)",
     )
+    parser.add_argument(
+        "--nondialogue-categories",
+        nargs="*",  # 0 or more
```

Comment:
Currently, this consumes all arguments given until the next option/flag (or end of command), i.e. it also consumes the filename if given as the immediately following argument:
```
$ ./qc2md.py -d "test/magilumi_02 - Dialogue.ass" --standalone-categories foo bar "test/[QC]_blank_petzku.txt" 
usage: qc2md [-h] [-r] [-c] [-d DIALOGUE] [-f {full,text}] [-F] [-T] [--pick-refs | --no-pick-refs] [--nondialogue-categories [NONDIALOGUE_CATEGORIES ...]]
             [--standalone-categories [STANDALONE_CATEGORIES ...]] [-o OUTPUT]
             filename
qc2md: error: the following arguments are required: filename
```

Unfortunately, there doesn't seem to be a great way to avoid this on the developer side: https://stackoverflow.com/a/26986546/4611644
Though users can easily work around this by using `--` to separate positional arguments from options:
```
$ ./qc2md.py -d "test/magilumi_02 - Dialogue.ass" --standalone-categories foo bar -- "test/[QC]_blank_petzku.txt"
```

One option would be using `action="append"` instead, but that would require specifying the option before *each* argument. Also, default handling would need to work differently (the argparse default value should be empty/left out, and we should instead check during the program if it is non-empty; if it is, we should use the set of defaults)